### PR TITLE
Cost analysis of a council election full block

### PR DESCRIPTION
**Pull request by kianenigma**
Current value is `166_000 DOLLARS`. With the current value of DOT, this is more than a million. But I expect the DOT to DOLLARS ratio to be tuned soon, therefore let's assume DOLLARS is actually 1USD. 

In this case, I think it is a bit too low. Current `VotingBaseBond` is `20 * DOLLARS + ...` only for the storage item. I would say we increase this to 50 or 100 DOLLARS to reach a safety net of half a million DOLLARS at least.

### CODE REVIEW

**Comment by shawntabrizi** [comment]
Just as a note, I think we want to deprecate the use of "DOLLAR" in our code base since it implies things which you mention clearly do not apply above.

It might make sense to come up with another unit of measurement. Maybe # of Big Macs?

https://www.economist.com/big-mac-index

**Comment by bkchr** [comment]
An attack on Polkadot costs 500 Big macs :D

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
@kianenigma some unused warning

**Review by Az111z — APPROVED** [review]

**Comment by kianenigma** [comment]
> Just as a note, I think we want to deprecate the use of "DOLLAR" in our code base since it implies things which you mention clearly do not apply above.
> 
> It might make sense to come up with another unit of measurement. Maybe # of Big Macs?
> 
> https://www.economist.com/big-mac-index

I haven't had time to look into this yet, but the current measure is also not too bad. We've never changed these constants in the past, so I am a bit worried about the consequences there. But, assuming that they will have no consequence, and we can easily update them, I think this model is easy and simple enough. 

For example, the governance bodies can re-evaluate this once per quarter based on the mean value of DOT, and just change this one number. 

I'm curious to see what happens with #2451.

File: runtime/polkadot/src/lib.rs [hunk 1]
```diff
@@ -1512,6 +1512,38 @@ mod test_fees {
...
+		let cost = voters as Balance * (VotingBondBase::get() + 16 * VotingBondFactor::get());
+		let cost_dollars = cost / DOLLARS;
+		println!(
+			"can support {} voters in a single block for council elections; total bond {}",
+			voters,
+			cost_dollars,
+		);
+		assert!(cost_dollars > 150_000); // DOLLAR ~ new DOT ~ 10e10
+	}
+
```
Comment (kianenigma) [inline runtime/polkadot/src/lib.rs:1546]:
```suggestion

#[test]
```

File: runtime/polkadot/src/lib.rs [hunk 2]
```diff
@@ -1512,6 +1512,39 @@ mod test_fees {
...
+#[test]
```
Comment (kianenigma) [inline runtime/polkadot/src/lib.rs:1547]:
```suggestion
	#[test]
```

**Comment by kianenigma** [comment]
bot merge

**Comment by bkchr** [comment]
@kianenigma you need to merge master

**Comment by kianenigma** [comment]
bot merge

**Comment by kianenigma** [comment]
> @kianenigma you need to merge master

@bkchr you need to approve

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
> > @kianenigma you need to merge master
> 
> @bkchr you need to approve 

Done 😜

**Pull request merged**